### PR TITLE
Release 0.13.54

### PR DESCRIPTION
**Pull request by cocreature**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/support/release-notes.rst [hunk 1]
```diff
@@ -6,6 +6,25 @@ Release notes
...
+
+0.13.54 - 2020-02-20
+--------------------
+
+Sandbox
+~~~~~~~
+
+- Removed the warnings regarding static time and scenarios on
+  initialization. We will not deprecate these until we have a stable
+  path forward.
```
Comment (bame-da) [inline docs/source/support/release-notes.rst]:
```suggestion
  alternative.
```

**Comment by SamirTalwar-DA** [comment]
Wait, don't we need to bump the number in _VERSION_?

**Comment by cocreature** [comment]
:man_facepalming:

**Comment by cocreature** [comment]
All manual tests passed

**Pull request merged**